scripts/update_e2e_coverage.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Updates coverage of TensorFlow e2e tests on all backends.
+
+Example usage: python3 update_e2e_coverage.py build-docs
+"""
+
+import argparse
+import collections
+import os
+import subprocess
+
+
+REFERENCE_BACKEND = 'tf'
+# Assumes that tests are expanded for the tf_also, iree_vmla, iree_llvmjit and
+# iree_vulkan backends.
+BACKENDS_TO_TITLES = collections.OrderedDict([
+    ('tf_also', 'tensorflow'),
+    ('iree_vmla', 'vmla'),
+    ('iree_llvmjit', 'llvm-ir'),
```

Comment:
llvmaot should also go here soon (if we want to differentiate between llvmjit and llvmaot... otherwise may rename to just "llvm"?)

no action needed, but could be worth checking that the UI can scale to adding another backend

Reply:
Adding another backend currently makes the tables scroll. This can be changed by overriding the tables' [`min-width`](https://github.com/pmarsceill/just-the-docs/blob/171ff71a2b13bb7e03f509159d3c9e31320ab4e0/_sass/tables.scss#L25), but the exact value of the override depends on what the new name is. (For example, adding a backend called `backend`, and then "setting `min-width` to the maximum value that doesn't cause the table to scroll" gives an override of `113px`). Longer backend names are more difficult to include without making the table look a bit wonky.

Hopefully there's a better way to do that that's less adhoc.